WordPress/openverse
# Update Load more button

## Fixes



Related to #2078 by @fcoveram 

## Description


This PR updates the Load more button to match the Figma mockups (see the links in Testing Instructions).

The button uses `filled-gray` variant. The `size` prop is set to `disabled` to add a custom size.

## Testing Instructions


Look at the "Load more" button at the bottom of the search results. It should match the Figma mockups:

[Component specs](https://www.figma.com/file/q4agvQxXI0ywTjwl36qkMe/Customized-buttons-update?type=design&node-id=912-9785&t=8aVsjj4iJVqxieJY-4) in the Figma file
 [Figma component](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?type=design&node-id=4799-12417&t=cohVFMFy7ESupLVR-4) in Design Library

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
